web/src/assets/styles/patternfly-overrides.scss

```diff
@@ -225,3 +225,9 @@ table td > .pf-v5-c-empty-state {
     --pf-v5-c-expandable-section__content--PaddingLeft: var(--spacer-normal);
   }
 }
+
+.pf-v5-c-form__group-label-help {
```

Comment:
Can it affect some other dialogs? I just afraid of some side effects...

Reply:
As far as I was able to check, currently it is not affecting. Anyway, I would like to discuss it with @dgdavid. Maybe we need some custom components for unitying  that info tips. For example, the icon in the installer language selector is not wrapped by a clickable element (e.g., button).